tests/integ/sagemaker/workflow/test_tuning_steps.py

```diff
@@ -177,11 +181,26 @@ def test_tuning_single_algo(
             create_arn,
         )
 
-        execution = pipeline.start(parameters={})
-        assert re.match(
-            rf"arn:aws:sagemaker:{region_name}:\d{{12}}:pipeline/{pipeline_name}/execution/",
-            execution.arn,
-        )
+        for _ in retries(
```

Comment:
nit: Thanks for adding retries, do we feel the need to add retries for other integ tests added above?